src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -344,6 +344,31 @@ public abstract partial class Attribute
         public virtual bool IsDefaultAttribute() { return default(bool); }
         public virtual bool Match(object obj) { return default(bool); }
         public virtual object TypeId { get { return default(bool); } }
+        public virtual object TypeId { get { return default(object); } }
```

Comment:
oops my bad did not pay close attention when resolving the merge conflict, will fix it